src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../interfaces/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    data-id={todo.id}
+    className={cn('TodoInfo', {
+      'TodoInfo--completed': todo.completed,
+    })}
+    key={todo.id}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    <UserInfo user={todo.user} />
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. The 'todo.user' object should be prepared in the App component and passed down to 'UserInfo' rather than importing it in this non-root component. Refactor the code to pass the 'user' object as a prop from the App component to maintain a single source of truth and better data flow control.